### ISSUE TITLE
[Accessibility] Add an option to allow Alt+F5 to jump to the next --word-diff instead of the whole line

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#166472]
CC @isidorn and @meganrogge

Although Alt+F5 and F7 provide accessible diff navigation for screen reader users, it is still challenging to identify exact changes when focused on the whole line.

Could you please consider adding a setting option to allow Alt+F5 to land in `--word-diff` instead of the whole line so that screen reader users can more easily identify modified locations?

## System Info

Version: 1.74.0-insider (system setup)
Commit: edccbd1307ef8055f26778c5a86f9eef885bc6ff
Date: 2022-11-16T05:24:12.776Z
Electron: 19.1.3
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes

**Comment by meganrogge** [comment]
Is your thought that for a line with 3 words changed, pressing `Alt+f5` would place the cursor at the start of word 1, start of word 2, start of word 3, start of word 1, and so on?

**Comment by jooyoungseo** [comment]
@meganrogge -- Exactly! As you may understand, good accessibility is minimizing cognitive load burden. That kind of navigation could improve blind programmers' productivity because we could save more time and cognitive effort and precisely get focused on exact changes upon the key press. Otherwise, we have to read through and compare between all the before and after diffs line by line, including something remaining the same.

**Comment by jooyoungseo** [comment]
For example, personally I have found the following options so useful when diffing changes in terminal:

``` bash
# Display word-based changes
git diff --word-diff

# character-based changes
git diff --word-diff-regex="."
```

The level of customization of Alt+F5 landing point would also contribute to general usability of `VSCode` for sighted users as well.

However, I think blind people are more disfranchised from the benefit of git diff because line-by-line diff takes much more time for us than people imagine since we cannot take a *glance*.

**Comment by VSCodeTriageBot** [comment]
This feature request is now a candidate for our backlog. The community has 60 days to [upvote](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
This feature request has not yet received the 20 community [upvotes](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) it takes to make to our backlog. 10 days to go. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
:slightly_frowning_face: In the last 60 days, this feature request has received less than 20 community upvotes and we closed it. Still a big Thank You to you for taking the time to create this issue! To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by jooyoungseo** [comment]
Related issue: #103285 as well as [this Stack Overflow question](https://stackoverflow.com/questions/36066118/setting-custom-git-diff-command-word-diff-in-visual-studio-code).

**Comment by VSCodeTriageBot** [comment]
:slightly_frowning_face: In the last 60 days, this feature request has received less than 20 community upvotes and we closed it. Still a big Thank You to you for taking the time to create this issue! To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
:slightly_frowning_face: In the last 60 days, this feature request has received less than 20 community upvotes and we closed it. Still a big Thank You to you for taking the time to create this issue! To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by meganrogge** [comment]
@hediet I discussed your thought about how we can present word diffs in an accessible way and @jooyoungseo mentioned this issue, which would fix that.

**Comment by meganrogge** [comment]
@hediet, JooYoung suggested that once we have implemented this, it would be great if on show of the hover, we could indicate if the change was an insertion, deletion, or modification. Would that be possible?

**Comment by hediet** [comment]
> it would be great if on show of the hover, we could indicate if the change was an insertion, deletion, or modification. Would that be possible?

I think it would be possible! But is certainly some work that requires planning.
